frontend/app/src/people/main/FocusView.tsx

```diff
@@ -186,6 +186,10 @@ function FocusedView(props: FocusViewProps) {
           newBody.description = description;
         }
 
+        if (newBody.price) {
+          newBody.price = Number(newBody.price);
+        }
```

Comment:
This fixes the issue of the bounty price being a string right? not sure if we want to return as a number from the backend so the frontend doesn't have to do this. But that may be a larger change